src/ansys/templates/python/common/{{cookiecutter.__project_name_slug}}/.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    {%- if cookiecutter.__template_name in ["pybasic", "pyansys"] %}
+    directory: "/"
+    {%- else -%}
+    directory: "/requirements"
+    {%- endif -%}
+    schedule:
+      interval: "daily"
+    labels:
+      - "maintenance"
+      - "dependencies"
```

Comment:
Oh, this is great! So now we are adding dependabot as part of the templates?

Reply:
Exactly, only for those which include some sort of CI. These are "pyansys-advanced" and all "pyace-*" ones.